main.go

```diff
@@ -164,8 +165,10 @@ func main() {
 		panic(util.Exit{0})
 	}
 
-	// Checks if Docker requires sudo access. Prints error message if so.
-	util.CheckSudo()
+	// Checks if we have elevated privileges to run docker. Only required for windows. Prints error message if privileges are missing.
+	if runtime.GOOS == "windows" {
```

Comment:
Nah, at this point if we ARE running windows, we default to prev. behavior, checking for access to docker and return an error if we can't. if we are NOT running windows, we don't care right now because we are going to check later and then attempt to use sudo if we can't.

Reply:
👍